internal/provider/resource_tfe_variable.go

```diff
@@ -717,6 +793,58 @@ func (r *resourceTFEVariable) ImportState(ctx context.Context, req resource.Impo
 	resp.Diagnostics.Append(diags...)
 }
 
+type replaceValueWOPlanModifier struct{}
+
+func (v *replaceValueWOPlanModifier) Description(ctx context.Context) string {
+	return "The resource will be replaced when the value of value_wo has changed"
+}
+
+func (v *replaceValueWOPlanModifier) MarkdownDescription(ctx context.Context) string {
+	return v.Description(ctx)
+}
+
+func (v *replaceValueWOPlanModifier) PlanModifyString(ctx context.Context, request planmodifier.StringRequest, response *planmodifier.StringResponse) {
```

Comment:
I *think* this can be replaced by an attribute-focused `RequiresReplaceIf`.

Reply:
[Here is what is looks like in my PR](https://github.com/hashicorp/terraform-provider-tfe/blob/f5b62f85ef39042a97627837338c1d9b9c6e42f1/internal/provider/resource_tfe_policy_set_parameter.go#L146-L169). The update case seems to be working ok, does it look like I am [missing any test coverage](https://github.com/hashicorp/terraform-provider-tfe/blob/f5b62f85ef39042a97627837338c1d9b9c6e42f1/internal/provider/resource_tfe_policy_set_parameter_test.go#L137-L200) to you?